frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -312,6 +319,73 @@ describe("WMG tissue auto-expand", () => {
     await removeCellFilter(page);
     await checkTissues(page, EXPECTED_FILTERED_TISSUES_WITH_DATASET_FILTER);
   });
+
+  test("Share link with genes and cellTypes", async ({ page }) => {
+    await Promise.all([
+      /**
+       * (thuang): This test asserts that the app does use the cellTypes passed
+       * in the share link in a `/filters` request.
+       * If this `waitForRequest` times out, it's likely because the app is NOT
+       * sending a request with the cellTypes passed in the share link.
+       */
+      page.waitForRequest(
```

Comment:
passing cellTypes in share link actually doesn't result in making `/filters` query with cellTypeIds in it, so this test will ensure we don't have the regression going forward

Reply:
thank you for adding tests 🙌🏽